app/controllers/autocompleted_fields_controller.rb

```diff
@@ -1,22 +1,27 @@
 class AutocompletedFieldsController < ApplicationController
 
   def update_by_project
-
     project = Project.find(params[:autocomplete_pattern][:project_id])
-    AutocompletedField.where(project_id: project.id).delete_all
+    if !params["separator"].present?
```

Comment:
Here you can simplify by using the "blank?" method:
params["separator"].blank?
is easier to read and equivalent to 
!params["separator"].present?

Reply:
Ok, thank you !